src/main/java/at/hannibal2/skyhanni/config/features/mining/PowderChestTimerConfig.java

```diff
@@ -0,0 +1,72 @@
+package at.hannibal2.skyhanni.config.features.mining;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorColour;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorSlider;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class PowderChestTimerConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Enable the feature.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = true;
+
+    @Expose
+    @ConfigOption(name = "Highlight Chests", desc = "Highlight chest with a color depending on how much time left until it despawn.")
+    @ConfigEditorBoolean
+    public boolean highlightChests = true;
+
+    @Expose
+    @ConfigOption(name = "Use Static Color", desc = "Use a single color for the chest highlight instead of chaning it depending of the time.")
```

Comment:
```suggestion
    @ConfigOption(name = "Use Static Color", desc = "Use a single color for the chest highlight instead of changing it depending of the time.")
```